OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -144,6 +144,9 @@ protected virtual AttackStatus TickAttack(Actor self, AttackFrontal attack)
 			if (!target.IsValidFor(self))
 				return AttackStatus.UnableToAttack;
 
+			if (attack.IsTraitPaused)
```

Comment:
Unfortunately this isn't quite right either :cry:

Returning `AttackStatus.UnableToAttack` makes the actor drop the target completely, which takes us from one extreme to the other. Ignoring `attack.IsTraitPaused` completely leaves us back at square one, with the bug that this PR originally set out to solve.

I believe the best workaround here is to put a `if (!attack.IsTraitPaused)` around the `CheckFire` calls below, and I will force-push over this PR to add this.